Replicator/tests/ReplicatorCollectionSGTest.cc

```diff
@@ -849,6 +849,148 @@ TEST_CASE_METHOD(ReplicatorCollectionSGTest, "Resolve Conflict SG", "[.SyncServe
     }
 }
 
+TEST_CASE_METHOD(ReplicatorCollectionSGTest, "Auto Purge Enabled - Revoke Access - SGColl", "[.SyncServerCollection]") {
+    const string idPrefix = timePrefix();
+    const string docIDstr = idPrefix + "apera-doc1";
+    const string channelIDa = idPrefix + "a";
+    const string channelIDb = idPrefix + "b";
+
+    _authHeader = SGUserAuthHeader;
+
+    // Create a temporary user for this test
+    HTTPStatus status;
+    C4Error error;
+    sendRemoteRequest("POST", "_user", &status, &error, R"({"name":"purgeRevoke","password":"password"})", true);
```

Comment:
There is helper for this purpose, createTestUser, which is meant for user per test. Have. you rebased to the latest master?

Reply:
Ah yeah, I forgot about that function. Will amend now.